app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -221,7 +221,7 @@ protected void onNewIntent(final Intent intent) {
     void loadFromIntent(final Intent intent) {
         final Uri uri = intent.getData();
         if (SessionStore.get().getCurrentSession() == null) {
-            String url = (uri != null ? uri.toString() : SessionStore.DEFAULT_URL);
+            String url = (uri != null ? uri.toString() : SessionStore.get().getHomeUri());
```

Comment:
I thought about just changing `SessionStore#loadUri` to accept a null `uri` and keeping the conditional logic in that method. would make things a tad more DRY. thoughts?